data/templates/frr/babeld.frr.j2

```diff
@@ -0,0 +1,85 @@
+{% from 'frr/distribute_list_macro.j2' import render_distribute_list %}
+{% from 'frr/ipv6_distribute_list_macro.j2' import render_ipv6_distribute_list %}
+!
+{# Interface specific configuration #}
+{% if interface is vyos_defined %}
+{%     for iface, iface_config in interface.items() %}
+interface {{ iface }}
+{%         if iface_config.type is vyos_defined('wired') or iface_config.type is vyos_defined('wireless') %}
+ babel {{ iface_config.type }}
+{%         endif %}
+{%         if iface_config.split_horizon is vyos_defined("enable") %}
```

Comment:
CLI also comes with the `split-horizon` auto option - what's it's use?

Reply:
The default value in FRR is `enable` for wired interfaces and `disable` for wireless interface. I just set `auto` as the default value. Maybe we should remove `auto` option and use the FRR default when this option is not set.